dev-lang/rust/rust-1.82.0.ebuild

```diff
@@ -307,6 +306,14 @@ src_prepare() {
 	#	clear_vendor_checksums "${i}"
 	#done
 
+	# Rust baselines to Pentium4 on x86, this ptach lowers the baseline to i586 when sse2 is not set.
```

Comment:
```suggestion
	# Rust baselines to Pentium4 on x86, this patch lowers the baseline to i586 when sse2 is not set.
```

Reply:
Thanks! I also misspelt the patch name too...